### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by camiriveron**
## Description
This PR creates the Home page UI and shows the Google Map. Right after signing up the user gets a Welcome Page, after dismissing the user gets the Empty Home Page.

## Trello Reference
[As a user I should be able to see a map so I can easily identify targets in the future](https://trello.com/c/rAMTxDzo/3-3-as-a-user-i-should-be-able-to-see-a-map-so-i-can-easily-identify-targets-in-the-future)

### Welcome Home Page
screen shot 2018-11-16 at 12 45 02 pm

### Empty Home Page
screen shot 2018-11-16 at 12 44 32 pm

### CODE REVIEW

File: src/containers/HomePage.js [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
-  </div>
-);
+import { GOOGLE_MAPS_URL } from 'constants/constants';
 
-export default HomePage;
+const HomePage = ({ signedUp }) =>
+  <div className="grid-x">
+    <div className="cell small-12 medium-3 show-for-medium " >
+      {signedUp && <Welcome />}
+      {!signedUp && <HomeSideBar />}
```
Comment (peter7z) [inline src/containers/HomePage.js]: What about a ternary operator?
`{signedUp ? <Welcome /> : <HomeSideBar />}`

**Review by peter7z — APPROVED** [review]
Good job!

File: src/containers/HomePage.js (hunk 1 above)
Comment (camiriveron) [reply src/containers/HomePage.js]: Good call!

**Review by kevquincke — APPROVED** [review]
Nice and clear!!! Great PR! 😎

File: src/reducers/signupReducer.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import * as types from '../actions/actionTypes';
```
Comment (felipegarcia92) [inline src/reducers/signupReducer.js:1]: The thing with this new reducer is that, and I know it's not well explained or documented anywhere in this repo, `redux-react-session` already stores an `authenticated` boolean. It also persists it in Local Storage so it remains among sessions.

**Review by lunemmer — COMMENTED** [review]
Nice!!! 👍

File: src/reducers/signupReducer.js (hunk 2 above)
Comment (camiriveron) [reply src/reducers/signupReducer.js:1]: The intention for the `USER_SIGNED_UP` action is to show when the user first signs up (registers in the system) in order to show up a welcome message in the Home Page. Then, every time the user logs in he gets the regular Home Page. That's why the `authenticated` boolean in the session wasn't useful for this scenario. 
Thanks for the info, I know I will use this flag in the future :)

Comment (felipegarcia92) [reply src/reducers/signupReducer.js:1]: Makes sense! 👍

**Pull request merged**